Nucleation_model_histogram.py

```diff
@@ -87,20 +97,26 @@ def residual(t, sol_vec):
     
     # number of moles in the nucleus with radius r_crit
     N_crit = 4./3.*m.pi*r_crit**3./v_Li2O2 
-    """Is the division by 3 correct? If I'm guessing correctly, this is surface area, 4pi r_crit^2..."""
+    """Is the division by 3 correct? If I'm guessing correctly, this is surface area, 4pi r_crit^2...
+        I think it is correct because its the number of moles in the nucleus so it would volume / molar volar
+        Though it does bring up an interesting thought: a hemishpere volume is 2/3 so we might want to change it actually?
+        https://www.sciencedirect.com/science/article/pii/S0013468619315907 its in here after equation 34
+    """
     # J mol-1 // energy barrier of the nucleation
     dG_crit = phi*4./3.*m.pi*gamma_surf*r_crit**2 
     if N_crit <0:
         dG_crit = 0
 
-    """Make sure the units work out, here."""
+    """Make sure the units work out, here.
+    Units check out with disclaimer see ppt"""
     Z = m.sqrt(dG_crit/(phi * 3. * m.pi * RT * N_crit)) # - // Zeldovich factor
     V_crit = 4./3. * m.pi * r_crit**3 # m3 // Critical volume
     """TODO: incorporate max packing fraction for circles."""
-    N_sites = A/(m.pi*r_crit**2) # number of nucleation sites
+    N_sites = A/(m.pi*r_crit**2)*pack_frac # number of nucleation sites
     k_nuc= D_LiO2*(a_d**-2) #nucleation rate calculated based on the distance between particles
-    """Should k_B*T be R*T, here, since Del_G_Crit is in J/mol?"""
-    DN_Dt = k_nuc*N_sites*Z*m.exp(-dG_crit/RT)*1E-20
+    """Should k_B*T be R*T, here, since Del_G_Crit is in J/mol?
+    I think it should be K_b*T because K_b is J/K or is that what you were asking?"""
```

Comment:
That is what I was asking - `k_B` is J/K-molecule. `R = N_A*k_B`, and is J/K-mol.